frontend/js/restqueries.js

```diff
@@ -0,0 +1,82 @@
+try {
+	var staticURL = "http://ec2-34-208-19-115.us-west-2.compute.amazonaws.com:18080/character/";
+	var playerURL = "OtherTestChar";
+	var DnDObj = getUser();
+	//getJSONField("player");
+}
+catch (err) {
+	console.log(err);
+}
+//getJSONField("name");
+populateDNDFields();
```

Comment:
As I mention below regarding the use of global variables. It might be better if you change populateDNDFields() to take in the DnDObj, ie. populateDNDFields(DnDObj)

try to avoid global variables if you can.